src/styles/profile.css

```diff
@@ -6,18 +6,38 @@
 
 .reserved-rocket {
   border: 1px solid lightgrey;
-  padding: 3% 3%;
   margin: 0;
+  padding: 4%;
+  font-size: 20px;
 }
 
 .reserved-rockets-container {
   flex: 0.5;
 }
 
 .rocket-info {
-  width: 350px;
+  width: 95%;
+  border: 1px solid gray;
+  border-radius: 6px;
 }
 
 .mission-profile {
   flex: 0.5;
+  margin-left: 20px;
+}
+
+.list-container {
+  border: 1px solid gray;
+  border-radius: 6px;
+  padding: 0;
+}
+
+li {
+  list-style: none;
+}
+
+.mission-item {
+  border-bottom: 1px solid gray;
+  padding: 4%;
+  font-size: 20px;
```

Comment:
It would be great if you decrease the top and bottom padding. 
 padding : 2% 4%;

Reply:
Done